CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# CHANGELOG
+
+## [Unreleased]
+
+### Overview
+
+- Upgrades cosmos-sdk to `v0.47.3` LSM fork created by persistence, including [barberry](https://forum.cosmos.network/t/cosmos-sdk-security-advisory-barberry/10825) security fix
+- Migrates to cometbft `v0.37.2`
+- Upgrades ibc-go to `v7.1.0` LSM fork including [huckleberry](https://forum.cosmos.network/t/ibc-security-advisory-huckleberry/10731) security fix
+- Upgrades wasmd to `v0.40.2` LSM fork & wasmvm to `1.2.4` including [cherry](https://github.com/CosmWasm/advisories/blob/main/CWAs/CWA-2023-002.md) bugfix
+- Some SDK 47 things to keep in mind:
+  - The SDK version includes some key store migration for the CLI. Make sure you backup your private keys before testing this! You can not switch back to v45
+  - CLI: `add-genesis-account`, `gentx`, `add-genesis-account`, `collect-gentxs` and others are now under `genesis` command as parent
+  - CLI: `--broadcast-mode block` was removed. You need to query the result for a TX with `persistenceCore q tx <hash>` instead
+  - ...add more?
+- Upgrades persistence-sdk to `vx.x.x` which includes
+  - add POB for MEV
+  - adds IBC hooks
+  - adds PFM module
+  - adds Oracle module (not in use for now)
+  - removes ibc-hooker
```

Comment:
>ibc-hooker

it's `ibchooker` to not confuse with `ibc-hooks`

Reply:
made the changes